### ISSUE TITLE
watch variables not announced if they're persistent

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#223219]
1. add some watch variables to the debug view
2. reload the window
3. with a screen reader enabled, hit a breakpoint that causes the variable to change
4. 🐛 the change isn't announced

**Comment by hediet** [comment]
In this example:
```
for (let i = 0; i < 100; i++) {
    console.log('hello', i); // breakpoint here
}
```

When ever debugging continues and stops again, the screenreader reads the line twice.

**Comment by meganrogge** [comment]
> In this example:
> 
> ```
> for (let i = 0; i < 100; i++) {
>     console.log('hello', i); // breakpoint here
> }
> ```
> 
> When ever debugging continues and stops again, the screenreader reads the line twice. The changed variable i is read though.

That's something others have also noticed (and so did I) that occurred before this feature. https://github.com/microsoft/vscode/issues/223431
